### PR TITLE
Reworked LandingPage, major changes to Gameloop

### PR DESCRIPTION
**Pull request by Tarkan67**

### CODE REVIEW

File: components/Map/Map.js [hunk 1]
```diff
@@ -80,27 +82,33 @@ const Map = ({
...
-          handleAddMarker(marker);
-          handleDistance(marker);
-        } else if (clickCount === 1 && !checkAnswer) {
-          handleClickCount(clickCount - 1);
-          markerStore ? map.removeLayer(markerStore) : null;
-          const marker = L.marker([lat, lng], { Icon }).addTo(layerGroup);
-          handleAddMarker(marker);
-          handleDistance(marker);
+        if (distanceRight !== 2 && distanceRight !== 1) {
+          console.log("distanceRight", distanceRight);
```
Comment (flohip) [inline components/Map/Map.js:97]: console.log for debugging?

File: components/PointsDisplay/PointsDisplay.js [hunk 2]
```diff
@@ -3,21 +3,22 @@ import { useSession } from "next-auth/react";
 import useSWR from "swr";
 import styles from "./PointsDisplay.module.css";
 
-export default function PointsDisplay() {
+export default function PointsDisplay({ currentPicture }) {
   const { data: session } = useSession();
+  console.log("points display", currentPicture);
```
Comment (flohip) [inline components/PointsDisplay/PointsDisplay.js:8]: console.log()

File: components/PointsDisplay/PointsDisplay.js [hunk 3]
```diff
@@ -3,21 +3,22 @@ import { useSession } from "next-auth/react";
...
   const { data: user } = useSWR(`/api/user/${session?.user.id}`, {
     isPaused: () => !session?.user.id,
   });
 
-  if (session) {
-    <Button variant="contained" className={styles.PointsDisplay}>
-      {user ? user.points : "not logged in"}
-    </Button>;
-  }
+  // if (session) {
```
Comment (flohip) [inline components/PointsDisplay/PointsDisplay.js:14]: is the outcommented stuff still needed with the code below?

File: pages/level_1.js [hunk 4]
```diff
@@ -53,8 +53,11 @@ export default function Game({
...
   setAnimation,
+  reload,
+  setReload,
+  distanceRight,
+  setDistanceRight,
 }) {
-  const [distanceRight, setDistanceRight] = useState();
   const [open, setOpen] = useState(true);
   const { data: session } = useSession();
   console.log(distance);
```
Comment (flohip) [inline pages/level_1.js:63]: console.log()

File: pages/level_1.js [hunk 5]
```diff
@@ -91,18 +96,43 @@ export default function Game({
...
+  }
+
   function handleNextPicture() {
+    if (currentPicture.id === 5) {
+      routerLandingPage();
+    }
     setDistanceRight(undefined);
-    setCurrentPicture(locations[randomInteger(4)]);
+    setSubmitCount(submitCount + 1);
+    console.log("currentPicture", currentPicture.id);
```
Comment (flohip) [inline pages/level_1.js:121]: console.log()

File: pages/level_2.js [hunk 6]
```diff
@@ -0,0 +1,337 @@
...
+  Tooltip,
+} from "@mui/material";
+import PointsDisplay from "../components/PointsDisplay/PointsDisplay";
+import useSWR from "swr";
+import { cx } from "@emotion/css";
+import { useRouter } from "next/router";
+
+const MapEffect = ({ useMap }) => {
+  const map = useMap();
+  useEffect(() => {}, [map]);
```
Comment (flohip) [inline pages/level_2.js:25]: empty useEffect?

File: pages/level_2.js [hunk 7]
```diff
@@ -0,0 +1,337 @@
...
+  animation,
+  setAnimation,
+  reload,
+  setReload,
+  distanceRight,
+  setDistanceRight,
+}) {
+  const [open, setOpen] = useState(true);
+  const { data: session } = useSession();
+  console.log(distance);
```
Comment (flohip) [inline pages/level_2.js:61]: console.log()

File: pages/level_2.js [hunk 8]
```diff
@@ -0,0 +1,337 @@
...
+          method: "PATCH",
+          headers: { "content-type": "application/json" },
+          body: JSON.stringify({
+            $inc: {
+              points: Math.round(1000 * (1 / (distance ^ 0.9))),
+            },
+          }),
+        });
+        const data = await response.json();
+        console.log("data", data);
```
Comment (flohip) [inline pages/level_2.js:92]: console.log()

File: pages/level_2.js [hunk 9]
```diff
@@ -0,0 +1,337 @@
...
+    router.push("/");
+  }
+
+  function handleNextPicture() {
+    if (currentPicture.id === 5) {
+      routerLandingPage();
+    }
+    setDistanceRight(undefined);
+    setSubmitCount(submitCount + 1);
+    console.log("currentPicture", currentPicture.id);
```
Comment (flohip) [inline pages/level_2.js:119]: console.log()

**Review by flohip — APPROVED** [review]

File: components/PointsDisplay/PointsDisplay.js (hunk 3 above)
Comment (Tarkan67) [reply components/PointsDisplay/PointsDisplay.js:14]: forgot that, thank you!

**Pull request merged**